hack/render-sync.sh

```diff
@@ -32,4 +32,8 @@ function rendersync() {
 
 rendersync manual-cluster/performance base/performance default
 rendersync bootstrap-cluster/performance pinned-cluster/default bootstrap/no-mcp
-rendersync bootstrap-cluster/performance pinned-cluster/default bootstrap-cluster/extra-mcp bootstrap/extra-mcp
\ No newline at end of file
+rendersync bootstrap-cluster/performance pinned-cluster/default bootstrap-cluster/extra-mcp bootstrap/extra-mcp
+rendersync base/performance manual-cluster/performance no-ref
```

Comment:
the `no-ref` set should not include any reference label, we need to parametrize the render to deal with this.
I'm not sure why we are adding the render-sync invocations in this PR btw.

Reply:
I will issue a different PR that would parametrize the render to deal with this.